src/Microsoft.AspNetCore.Rewrite/UrlRewrite/UrlRewriteExtensions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+
+namespace Microsoft.AspNetCore.Rewrite.UrlRewrite
```

Comment:
Put this extension in the base Microsoft.AspNetCore.Rewrite namespace so you don't have to add it in startup.